src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/commands/change_point/fields_suggestions_after.test.ts

```diff
@@ -0,0 +1,262 @@
+            text: 'count',
+            incomplete: false,
+            type: 'identifier',
+          },
+        ],
+        location: {
+          min: 143,
+          max: 147,
+        },
+        text: 'count',
+        incomplete: false,
+        parts: ['count'],
+        quoted: false,
+        name: 'count',
+        type: 'column',
+      },
+      {
+        name: 'on',
+        args: [
+          {
+            args: [
+              {
+                name: 'field1',
+                location: {
+                  min: 152,
+                  max: 160,
+                },
+                text: 'field1',
+                incomplete: false,
+                type: 'identifier',
+              },
+            ],
+            location: {
+              min: 152,
+              max: 168,
+            },
+            text: 'field1',
+            incomplete: false,
+            parts: ['field1'],
+            quoted: false,
+            name: 'field1',
+            type: 'column',
+          },
+        ],
+        location: {
+          min: 149,
+          max: 168,
+        },
+        text: '',
+        incomplete: false,
+        type: 'option',
+      },
+      {
+        name: 'as',
+        args: [
+          {
+            args: [
+              {
+                name: 'changePointType',
+                location: {
+                  min: 173,
+                  max: 187,
+                },
+                text: 'changePointType',
+                incomplete: false,
+                type: 'identifier',
+              },
+            ],
+            location: {
+              min: 173,
+              max: 187,
+            },
+            text: 'changePointType',
+            incomplete: false,
+            parts: ['changePointType'],
+            quoted: false,
+            name: 'changePointType',
+            type: 'column',
+          },
+          {
+            args: [
+              {
+                name: 'pValue',
+                location: {
+                  min: 190,
+                  max: 195,
+                },
+                text: 'pValue',
+                incomplete: false,
+                type: 'identifier',
+              },
+            ],
+            location: {
+              min: 190,
+              max: 195,
+            },
+            text: 'pValue',
+            incomplete: false,
+            parts: ['pValue'],
+            quoted: false,
+            name: 'pValue',
+            type: 'column',
+          },
+        ],
+        location: {
+          min: 170,
+          max: 195,
+        },
+        text: '',
+        incomplete: false,
+        type: 'option',
+      },
+    ],
+    location: {
+      min: 130,
+      max: 195,
+    },
+    text: 'CHANGE_POINTcountONextension.keywordASchangePointType,pValue',
+    incomplete: false,
+    type: 'command',
+  } as unknown as ESQLAstCommand;
+
+  it('should return the correct fields after the command without AS option', () => {
```

Comment:
nit: for future eyes looking at this, would be good to explicitly say what it does, instead of "...returns the correct ...", i.e what is correct?

```suggestion
  describe('CHANGE_POINT', () => {
    it('adds "type" and "pvalue" fields, when AS option not specified', () => {
```

Reply:
Makes sense, done here https://github.com/elastic/kibana/pull/218242/commits/17281a82f2ef2347e5a8acceacec314b0077b9ae